### PR TITLE
add example repo on using neon database with authjs

### PR DESCRIPTION
**Pull request by ananis25**
Companion repo for https://github.com/neondatabase/website/pull/2112

### CODE REVIEW

**Comment by rishi-raj-jain** [comment]
Hey @ananis25,

Can you do a quick review per https://github.com/neondatabase/examples/blob/main/CONTRIBUTING.md?

**Comment by ananis25** [comment]
Thanks for looking, done.

File: auth/with-authjs-next/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "with-authjs-next",
+  "version": "0.1.0",
+  "private": true,
```
Comment (rishi-raj-jain) [inline auth/with-authjs-next/package.json:4]: Can we remove private and version since this is not a package?

**Review by rishi-raj-jain — COMMENTED** [review]
If you get additional time to add prettier, it'd be awesome.

**Comment by ananis25** [comment]
Thanks, done.

**Pull request merged**